playbooks/preflight-check.yml

```diff
@@ -138,3 +138,22 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1371 detect if enough space for rootfs backup
+- name: Check space for rootfs backup on infras
+  hosts: "os-infra_hosts"
+  user: root
+  tags:
+    - rootfs_backup_space_check
+  tasks:
```

Comment:
These tasks can be added to the above to the https://github.com/rcbops/rpc-upgrades/blob/master/playbooks/preflight-check.yml#L68-L88 playbook and limited using a "when" for hosts in the "os-infra_hosts" group.


example:

``` yaml
- name: Gather filesystem used percentages
  shell: |
    df -h | tail -n +2 | sed s/%//g | awk '{if($5 > 50) print "Alert: "$5 "% full - "$0}'
  register: lxc_fs_check
  when:
    - inventory_host in groups["os-infra_hosts"]

- name: Check if any filesystem is greater than 50% full
  debug:
    msg: >-
      The filesystem(s) found to be (nearly) full.
      Details: {{ item.split() | join(' ') }}
  failed_when: true
  when:
    - inventory_host in groups["os-infra_hosts"]
    - lxc_fs_check.stdout_lines
  with_items: "{{ lxc_fs_check.stdout_lines }}"
```

Reply:
If we're only concerned with the LXC file system we should also do some introspection on the `lxc` volume group if present to ensure it has enough extents to create the container clones.